js/adapt-contrib-hotgraphic.js

```diff
@@ -159,16 +159,16 @@ define(function(require) {
             var $currentHotSpot = this.$('.' + $(event.currentTarget).data('id'));
             $currentHotSpot.show().addClass('active');
             
-            var currentIndex = this.$('.hotgraphic-item.active').index();
+            var currentIndex = $(event.currentTarget).data('index');
             this.setVisited(currentIndex);
             
-            this.openPopup();
+            this.openPopup(currentIndex);
            
             this.applyNavigationClasses(currentIndex);
         },
         
-        openPopup: function() {
-            var currentIndex = this.$('.hotgraphic-item.active').index();
+        openPopup: function(index) {
+            var currentIndex = index || this.$('.hotgraphic-item.active').index();
```

Comment:
can the latter half of the || vanish?

Reply:
can first half of the || vanish and directly use index.

Reply:
It's a good point. We don't need to redefine the index as currentIndex or use the dom.index() function at all here.